README.md

```diff
@@ -0,0 +1,85 @@
+## TAiO
+
+University course's project, provides CLI for multigraph (graph theory) comparison.
+
+### Getting started
+TODO (dependencies, how to install on different OSes)
+#### Nix
+There is flake prepared, so it is enough to enter development environment:
+```bash
+nix develop
+```
+### Compilation
+#### GNU/Linux
+**git**
+
+First clone repository, then
+```bash
+git submodule update --init --recursive
+```
+**CMake**
+
+We recommend:
+```bash
+mkdir build
+cd build
+cmake .. && cmake --build .
+```
+but similar result can be obtained in many ways using CMake CLI.
+
+### Command Line Interface
+```bash
+./multigraph_comparator [OPTIONS] file1 file2
+```
+
+Compares multigraphs.
+
+#### Positionals
+  - `file1` Path to the first multigraph file
+  - `file2` Path to the second multigraph file
+
+#### Options
+  - `-h,--help`              Print this help message and exit
+  - `-i,--index1` uint       Index of the multigraph in the first file (default: 0)
+  - `-j,--index2` uint       Index of the multigraph in the second file (default: 0)
+
+#### Example
+```bash
+./multigraph_comparator file1.txt file1.txt -i 0 -j 1
+./multigraph_comparator file1.txt file2.txt
+```
+
+### Input format
+Multigraphs are saved using their adjacency matrix in following format:
+```
+N <- count of multigraphs saved in file
```

Comment:
Multigraphs should be just listed in the file, and we should read to the end of file, becouse homenda will test other inputs on other programms and that's the structure he wants

Reply:
That's interesting, because he did specified the exact format he requires in one email:
```
2. Dane wejściowe będą zapisane w formacie wspólnym dla wszystkich zespołów z założeniem wymienialności danych między zespołami. Będą to pliki tekstowe zawierające opisy grafów (jednego lub więcej) oddzielone pustą linią. Pierwszy wiersz pliku zawiera liczbę grafów zapisanych w pliku. Od drugiego wiersza zapisane są opisy grafów:
- pierwszy wiersz opisu zawiera liczbę wierzchołków, ta informacja jest zapisana w jednym wierszu pliku,
- następne wiersze pliku zawierają wiersze macierzy sąsiedztwa z elementami oddzielonymi spacją,
- po macierzy sąsiedztwa w kolejnych wierszach pliku mogą być zapisane dodatkowe dane.
W macierzy sąsiedztwa zapisane są liczby krawędzi między wierzchołkami (dla grafów to będzie 0 lub 1, dla multigrafów to będzie 0 lub dodatnia liczba całkowita). Dla grafów nieskierowanych macierz sąsiedztwa będzie symetryczna względem głównej przekątnej. Informację o rodzaju grafu można np. dodatkowo zapisać bezpośrednio po macierzy sąsiedztwa.
```
In the other email he just specified that he would accept solutions working only for single multigraph per file:
```
Przyjmijmy, że program umożliwi wykonanie obliczeń dla wejściowych plików tekstowych zwierających po jednym grafie. Zatem, dla zadania wyznaczenia odległości dwóch grafów, program musi umożliwić wczytanie dwóch takich plików. Wczytanie plików zawierających więcej niż jeden graf i ich obsługa jest dodatkową nieobowiązkową cechą. 
```
So, for me, it is not clear whether the format should change.

Did he say something else about it during online meeting with you?